### PR TITLE
[PR] Update structure and documentation for new project structure

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
I'll update this once #7 is merged.

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: README.md [hunk 1]
```diff
@@ -30,12 +30,9 @@ git status
 
 There are several places throughout the WSUWP Plugin Skeleton code that should be changed to match the new plugin.
 
-1. The main plugin file, `wsuwp-plugin-skeleton.php`, should be renamed to `wsuwp-new-feature.php`.
-1. The main class file, `includes/class-wsuwp-plugin-skeleton.php`, should be renamed to `includes/class-wsuwp-new-feature.php`.
-1. The main tests file, `tests/test-wsuwp-plugin-skeleton.php`, should be renamed to `tests/test-wsuwp-new-feature.php`.
-1. In all files, update uses of the class `WSUWP_Plugin_Skeleton` to be `WSUWP_New_Feature`.
-1. In `wsuwp-plugin-skeleton.php` and `tests/bootstrap.php`, be sure included files are named properly.
-1. Update the plugin headers in `wsuwp-new-feature.php` to provide the project name, description, authors, etc...
+1. The main `includes/` file, `includes/wsuwp-plugin-skeleton.php`, should be renamed to match the new plugin's slug.
```
Comment (philcable) [inline README.md:33]: I suggest adding a step about updating the namespace in the `includes/wsuwp-plugin-skeleton.php` file, too.

**Review by philcable — APPROVED** [review]
👍  Nice! I left one inline comment, otherwise good to go!

File: README.md (hunk 1 above)
Comment (jeremyfelt) [reply README.md:33]: Good call!

**Pull request merged**